app/assets/javascripts/Patient.js

```diff
@@ -24,9 +26,7 @@ const PatientSchema = DataElementSchema({
   // Cypress use a common field, it should be made a field on this model,
   // and not put into extendedData.
   extendedData: {},
-}, { id: false });
```

Comment:
Why is this line getting deleted? We definitely need it for accessing the Patient Ids with a method.